tests/main/snap-on-non-shared-root/task.yaml

```diff
@@ -22,12 +22,12 @@ execute: |
     test-snapd-tools.echo hello
 
     echo "Ensure we have a shared mount of $SNAPMOUNTDIR"
-    cat /proc/self/mountinfo |MATCH "$SNAPMOUNTDIR $SNAPMOUNTDIR.*shared:1"
+    cat /proc/self/mountinfo |MATCH "$SNAPMOUNTDIR $SNAPMOUNTDIR.*shared:[0-9]*"
```

Comment:
Could you please post the log of the error that this change fixes? I haven't seen this test failing yet, perhaps something is going on that needs more attention.